src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -426,11 +431,108 @@ const formConfig = {
               }
             }
           }
+        },
+        recordUpload: {
+          title: 'Upload your private medical records',
+          depends: (formData, index) => {
+            const hasRecords = _.get(`disabilities.${index}.view:privateMedicalRecords`, formData);
+            // TODO: enable once previous chapter merged
+            // const uploadRecords = _.get(`disabilities.${index}.view:uploadPrivateRecords`, formData);
+            const uploadRecords = true;
+            return hasRecords && uploadRecords;
+          },
+          path: 'supporting-evidence/:index/documents',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          uiSchema: {
+            disabilities: {
+              'ui:options': {
+                itemName: 'Private Medical Record'
+              },
+              items: {
+                privateRecords: Object.assign({},
+                  fileUploadUI('Upload your private medical records', {
+                    allowRename: true,
+                    itemDescription: 'Adding additional evidence:',
+                    endpoint: '/v0/preneeds/preneed_attachments', // TODO: update this with correct endpoint (e.g. '/v0/21-526EZ/medical_records')
```

Comment:
@kreek Any idea what this file upload endpoint will look like or when it'll be available? (Or is @omgitsbillryan the right person to ask?)